tests/integrationtest/t/executor/issues.test

```diff
@@ -779,3 +779,8 @@ select from_unixtime( if(col2 >9999999999, col2/1000, col2), '%Y-%m-%d %H:%i:%s'
 
 # TestIssue56641
 select ((exists (select 1)) * -5) as c1;
+
+# TestIssue61880
+CREATE TABLE test_61880 (c1 blob);
+insert into test_61880 (c1) values(0);
+select * from test_61880;
```

Comment:
I tyied with MySQL 8.0.42, the result is 0x30

Reply:
interesting will learn it and closing